Clarify doc on text_chunking processor: nested field in input and output (#8229)
opensearch-project/documentation-website: _ingest-pipelines/processors/text-chunking.md

| `description`	              | String	    | Optional	 | A brief description of the processor. |
| `tag`	| String	    | Optional	 | An identifier tag for the processor. Useful when debugging in order to distinguish between processors of the same type.	|

> If you wish to perform chunking on nested map values, please specify `input_field` and `output_field` in a map-like format and not use `.`. Instead of `"field_map": { "foo.bar": "foo.bar_chunk"}`, you should use `"field_map": { "foo": { "bar": "bar_chunk"} }`.

Reviewer: ```suggestion
To perform chunking on nested fields, specify `input_field` and `output_field` values as JSON objects. Providing a dot path of the nested field is not supported. For example, use `"field_map": { "foo": { "bar": "bar_chunk"} }` instead of `"field_map": { "foo.bar": "foo.bar_chunk"}`.
```

> To perform chunking on nested fields, specify `input_field` and `output_field` values as JSON objects. Providing a dot path of the nested field is not supported. For example, use `"field_map": { "foo": { "bar": "bar_chunk"} }` instead of `"field_map": { "foo.bar": "foo.bar_chunk"}`.

Reviewer 2: ```suggestion
To perform chunking on nested fields, specify `input_field` and `output_field` values as JSON objects. Dot paths of nested fields are not supported. For example, use `"field_map": { "foo": { "bar": "bar_chunk"} }` instead of `"field_map": { "foo.bar": "foo.bar_chunk"}`.
```
